index.ts

```diff
@@ -136,7 +136,7 @@ async function getStreamableImage(
       downscaleFactorBytesEstimated
     );
 
-    streamable = streamable.resize(Math.round(imageMeta.width * factor));
```

Comment:
I'm asking myself if this wouldn't ALWAYS rotate, so it would rotate the correctly rotated images in the wrong direction? Shouldn't there be an if-check or so?

Reply:
Valid concern. According to the sharp docs it states that "If no angle is provided, it is determined from the EXIF data" which seems to be exactly what we want here. 
https://sharp.pixelplumbing.com/api-operation

Reply:
Perfect, can you add this as a comment so it's not forgotten?